universe/ValueRef.cpp

```diff
@@ -1757,6 +1757,22 @@ double ComplexVariable<double>::Eval(const ScriptingContext& context) const
         return part_type->Capacity();
 
     }
+    else if (variable_name == "EmpireMeterValue") {
+        int empire_id = ALL_EMPIRES;
+        if (m_int_ref1)
+            empire_id = m_int_ref1->Eval(context);
+        Empire* empire = GetEmpire(empire_id);
+        if (!empire)
+            return -1.0;
+
+        std::string empire_meter_name;
+        if (m_string_ref1)
+            empire_meter_name = m_string_ref1->Eval(context);
+        Meter * meter = empire->GetMeter(empire_meter_name);
+        if (!meter)
+            return 0.0;
```

Comment:
I did not verify, but I think you could script an empire meter. So it could be that you ask for the value before the meter was created. For that I assumed a default value of 0 is sensible.
This is different from not finding the empire. That is certainly an error. I didnt see any error handling there so I returned a value that is probably recognizable as wrong (-1).